docs/siteconfig.rst

```diff
@@ -236,3 +236,11 @@ Here is a sample configuration with many of the options set and documented::
       api_token: your_api_token
       user_token: your_user_token
       machine_types: ['mira', 'smithi']
+
+    # Settings for https://github.com/ryg-/pelagos
```

Comment:
I guess we need to add push 'pelagos' to at least to 'suse' if not to 'ceph'

Reply:
link removed